store/ui.ts

```diff
@@ -14,7 +14,8 @@ export const uiStore = {
     submenu: {
       depth: false,
       path: []
-    }
+    },
+    supportsWebp: false
```

Comment:
it should be turned on by default, because in that case we will load on server image with `jpg` and on client it will be replaced by `webp`. Which will result in loading two images on all browsers that support webp.
So if we will have image in webp format on server and on client side then we will be wrong only in 13% browsers (thats the number of browsers that doesn't support webp). Which will also result in loading two images. First webp and then jpg.